lithium/lithium.py

```diff
@@ -134,6 +134,63 @@ def readTestcaseLine(self, line):
             self.parts.append(line[i:i + 1])
 
 
+class TestcaseJsStr(TestcaseChar):
+    atom = "jsstr"
+
+    def readTestcase(self, filename):
+        self._instr = None  # pylint: disable=attribute-defined-outside-init
+        self._chars = []  # pylint: disable=attribute-defined-outside-init
+
+        super(TestcaseJsStr, self).readTestcase(filename)
+
+        assert self._instr is None, "Unexpected EOF looking for end of string (%s)" % self._instr
+        del self._instr
```

Comment:
It would be nice to comment why there are del functions here in readTestcase but not in readTestcaseLine..